specs/commandline/sign.md

```diff
@@ -155,6 +157,20 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
+### Sign an OCI artifact and timestamp the signature with user specified timestamp authority
```

Comment:
We can explain TSA in one place, then refer to the short name TSA directly.

```suggestion
### Sign an OCI artifact and timestamp the signature with user specified RFC3161 Timestamp Authority (TSA)
```

Reply:
Updated.